indico/modules/categories/controllers/management.py

```diff
@@ -351,7 +353,15 @@ class RHDeleteEvents(RHManageCategorySelectedEventsBase):
     def _process(self):
         is_submitted = 'confirmed' in request.form
         if not is_submitted:
-            return jsonify_template('events/management/delete_events.html', events=self.events)
+            # find out which active bookings are linked to the events in question
+            num_bookings = (Reservation.query
+                            .join(ReservationLink)
+                            .join(Event, Event.id == ReservationLink.linked_event_id)
+                            .filter(Event.id.in_(map(attrgetter('id'), self.events)),
```

Comment:
`.in_(e.id for id in self.events)` is actually shorter than map+attrgetter

Reply:
I guess you mean `... for e ...` 😉 but yeah, thanks :)